### DISCUSSION TITLE
Automation / GPO / MSI

### DISCUSSION THREAD

**Discussion by ERR1R** [valinet/ExplorerPatcher#2379] [Ideas]
Hi I have  got some questions,

I'm working with task sequences  in SCCM and want to use explorer patcher to get installed as an application in the task sequence.
When I'm installing this (with a repackaged version (msi) in the task sequence) the login screen is flikkering ON/OFF and after a restart a couple of times settles down.   

- Is there any silent install switch i can use, maybe create an msi installer. I repackaged the tool but its better if you could provide one ~And also include the symbols so that we do not need to download symbols.~ 

 - Also, is it possible to get it installed globally with all settings, like editing NTUSER.DAT?  Im trying to edit NTUSER.DAT and import the register export but that doesn't seem to work for all settings. probably because those virtual register keys.  
 
 - Does the register import works if i would use a gpo in current user context or do i still need to configure it for every user that logs in? 

Problem is that every time a new user signs in we have to set the configuration and this basically makes this tool (sad) useless for automation.  

Thank you for reading and a great tool

**Comment by pyrates999** [comment]
New Symbols are released by Microsoft every time a new update for windows is released.  That's why symbols can never be included.  See here for more information: https://github.com/valinet/ExplorerPatcher/wiki/Symbols
